runtime/binding-sse/src/test/java/io/aklivity/zilla/runtime/binding/sse/internal/streams/server/DataIT.java

```diff
@@ -78,6 +78,16 @@ public void shouldReceiveMultipleMessages() throws Exception
         k3po.finish();
     }
 
+    @Test
+    @Configuration("server.validator.yaml")
+    @Specification({
+        "${net}/validation/request",
+        "${app}/validation/server" })
+    public void shouldValidateMessages() throws Exception
+    {
+        k3po.finish();
+    }
+
```

Comment:
We need a `shouldRejectMessages` too to prove validation is not just always allowing.

Reply:
Inside this test we have an invalid data as well. I just wanted to include it in the same test, as if the payload is invalid we don't close the stream. Or maybe put it in a seperate and assert for the invalid event?

Reply:
Yes, agreed. Please separate the positive validation and negative validation scenarios.